cast/src/main.rs

```diff
@@ -97,9 +104,8 @@ async fn main() -> Result<()> {
             )
             .await?;
 
-            // todo: #2107
-            eprintln!("Contract address: {contract_address}");
-            eprintln!("Transaction hash: {transaction_hash}");
+            print_formatted("Contract address: ", contract_address, cli.int_format);
```

Comment:
eprintln! redirects to stderr, where println! to stdout. I think we want to preserve outputting to stderr here.